src/server/rewards/passive/reportAccepted.reward.ts

```diff
@@ -0,0 +1,22 @@
+import { createBuzzEvent } from '../base.reward';
+
+const type = 'reportAccepted' as const;
+export const reportAcceptedReward = createBuzzEvent({
+  type,
+  description: 'For each report you make that is accepted',
+  awardAmount: 50,
+  caps: [{ amount: 1000, interval: 'month', keyParts: ['toUserId'] }],
```

Comment:
@JustMaier There's no easy way to actually make this an active reward atm, cause all `onDemand` rewards are reseted daily. So, I moved it to be passive but I believe we need to make adjustments to actually apply it, which I'm not familiar with. Any guidance you can give will help 🙏